src/AeroSharp/DataAccess/MapAccess/IMapOperator.cs

```diff
@@ -60,4 +61,21 @@ CancellationToken cancellationToken
     /// <param name="cancellationToken"> A cancellation token to cooperatively cancel the operation. </param>
     /// <returns> A task that represents the asynchronous operation of deleting the map. </returns>
     Task DeleteAsync(string recordKey, CancellationToken cancellationToken);
+
+    /// <summary>
+    ///    Asynchronously get {key: value} entry where map.value is the ith smallest value where i == rank, starting
+    ///    at zero being the lowest value.
+    /// </summary>
+    /// <param name="recordKey"> The key in the Aerospike record containing the map. </param>
+    /// <param name="bin"> The bin containing the map. </param>
+    /// <param name="rank"> The rank that we want to fetch, 0...n, -1 being the max. </param>
+    /// <param name="context"> The context within the map, if any. </param>
```

Comment:
Similar nit here. Let's keep the comments for this `context` param consistent across various usages so it doesn't lose meaning.